onnxruntime/core/providers/coreml/model/model.mm

```diff
@@ -247,210 +247,6 @@ Status CopyMLMultiArrayBuffer(const void* mlmultiarray_buffer, void* tensor_buff
 }
 }  // namespace
```

Comment:
now that we don't have an Objective-C class in the global namespace to deal with, could all of the preceding helper functions (in fact, most of this file) be put into the `onnxruntime::coreml` namespace? and then we can also remove the `using namespace` directives above.